src/account/reducer.ts

```diff
@@ -31,6 +31,7 @@ export interface State {
   accountToRecoverFromStoreWipe: string | undefined
   dismissedKeepSupercharging: boolean
   dismissedStartSupercharging: boolean
+  goldEducationCompleted: boolean
```

Comment:
Moved `state.goldToken.educationCompleted` to `state.account.goldEducationCompleted` since it doesn't have anything to do with the goldToken data and it allows us to delete the `goldToken` state from redux for good